integration/misc/packages/contracts/src/systems/ErrorTestSystem.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { System } from "@latticexyz/world/src/System.sol";
+import { Position } from "../CustomTypes.sol";
+
+contract ErrorTestSystem is System {
```

Comment:
Since the directory here infers that this is a "test" system, how about calling this `SystemWithCustomErrors`?

Reply:
changed, except so it ends with `System` to include it in worldgen by default

Reply:
oh right, I forgot about that requirement